build.gradle

```diff
@@ -495,6 +495,66 @@ project(':iceberg-hive-metastore') {
   }
 }
 
+project(':iceberg-hiveberg') {
```

Comment:
Are we modifying the root `build.gradle`? Why?

Reply:
Could you also clarify where these dependencies are copied from?

Reply:
We are moving classes from legacy package to new module here as demo. This modification is needed to add a new module. If we move them to a new code repo, this change can be removed. All the dependencies are from hive-metastore `compileOnly` and `testCompile` dependencies, which are necessary and cannot be pulled by pulling hive-metastore module